dev/tests/js/jasmine/tests/app/code/Magento/Customer/frontend/js/section-config.test.js

```diff
@@ -0,0 +1,239 @@
+                obj['Magento_Customer/js/section-config']({
+                    sections: {
+                        '*/subpath': [
+                            'section'
+                        ]
+                    },
+                    baseUrls: [
+                        'http://localhost.com/',
+                        'https://localhost.com/'
+                    ]
+                });
+
+                expect(obj.getAffectedSections('https://localhost.com/path/subpath')).toEqual(['section']);
+            });
+
+            it('Strips "index.php" suffix from provided URL.', function () {
+                obj['Magento_Customer/js/section-config']({
+                    sections: {
+                        'path': [
+                            'section'
+                        ]
+                    },
+                    baseUrls: [
+                        'http://localhost.com/'
+                    ]
+                });
+
+                expect(obj.getAffectedSections('http://localhost.com/path/index.php')).toEqual(['section']);
+            });
+
+            it('Adds sections for all URLs "*" to found ones.', function () {
+                obj['Magento_Customer/js/section-config']({
+                    sections: {
+                        'path': [
+                            'section'
+                        ],
+                        '*': [
+                            'all'
+                        ]
+                    },
+                    baseUrls: [
+                        'http://localhost.com/'
+                    ]
+                });
+
+                expect(obj.getAffectedSections('http://localhost.com/path')).toEqual(['section', 'all']);
+            });
+
+            it('Returns "*" sections for all URLs.', function () {
+                obj['Magento_Customer/js/section-config']({
+                    sections: {
+                        '*': [
+                            'all'
+                        ]
+                    },
+                    baseUrls: [
+                        'http://localhost.com/'
+                    ]
+                });
+
+                expect(obj.getAffectedSections('http://localhost.com/path')).toEqual(['all']);
+            });
+
+            it('Properly concatenates normal and "*" sections for all URLs.', function () {
+                obj['Magento_Customer/js/section-config']({
+                    sections: {
+                        'path': [
+                            'section'
+                        ],
+                        '*': [
+                            'all'
+                        ]
+                    },
+                    baseUrls: [
+                        'http://localhost.com/'
+                    ]
+                });
+
+                expect(obj.getAffectedSections('http://localhost.com/path')).toEqual(['section', 'all']);
+            });
```

Comment:
This test is the same as https://github.com/magento/magento2/pull/25764/files#diff-b2f763698c12c5411f66d86c6390b5adR99-R114

Reply:
True, will fix ASAP.

Reply:
Fixed.